lib/python/treadmill_aws/bootstrap/zookeeper/aws.linux/treadmill/bin/run_real.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+# This script is not in chroot yet, so need to resolve local directory.
+
+CHROOT={{ _alias.chroot }}
+CHMOD={{ _alias.chmod }}
+CHOWN={{ _alias.chown }}
+ECHO={{ _alias.echo }}
+GREP={{ _alias.grep }}
+LS={{ _alias.ls }}
+MKDIR={{ _alias.mkdir }}
+MOUNT={{ _alias.mount }}
+RM={{ _alias.rm }}
+
+unset KRB5CCNAME
+unset KRB5_KTNAME
+
+for SVC in $($LS {{ dir }}/treadmill/init); do
+    $GREP {{ dir }}/treadmill/init/$SVC/\$ {{ dir }}/.install > /dev/null
+    if [ $? != 0 ]; then
+        if [ -d {{ dir }}/treadmill/init/$SVC ]; then
+            $ECHO Removing extra service: $SVC
+            $RM -vrf {{ dir }}/treadmill/init/$SVC
+        fi
+    fi
+done
+
+$RM -vf {{ dir }}/treadmill/init/*/data/exits/*
+$RM -vf {{ dir }}/treadmill/tombstones/*
+
+for DIR in $(ls -a /); do
+    # Ignore . and .. directories
+    if [[ "${DIR}" != "." && "${DIR}" != ".." && -d /${DIR} ]]; then
+        $MKDIR -p {{ dir }}/${DIR}
+        if [ $DIR == "tmp" ]; then
+            # Make /tmp in chroot rw for all with sticky bit.
+            $CHMOD 1777 {{ dir }}/$DIR
+        else
+            $MOUNT -n --rbind /${DIR} {{ dir }}/${DIR}
+        fi
+    fi
+done
+
+cd {{ dir }}
+
+# Starting svscan
+export PATH={{ _alias.s6 }}/bin:$PATH
+
+$CHOWN -R {{ treadmillid }} {{ dir }}/treadmill/zookeeper/data
```

Comment:
[nit] I have the same problem and we need a better solution for this.

Reply:
put .owner file in the directory. like .owner -> {{ treadmillid }}. it is not recursive though, something to fix.

fixed in the code, and will remove chown.